user/how-to-guides/how-to-open-urls-in-other-qubes.md

```diff
@@ -1,174 +1,105 @@
 
 ### Using the `allow` action
 
-This action allows a given RPC service and source/destination qubes without
-prompting the user.
+This action allows a specified RPC service to be invoked between source and target qubes without displaying a confirmation prompt in dom0.
 
-When an `allow` action is defined for a destination other than `@dispvm`, the
-destination qube is the one given as an argument to `qvm-open-in-vm` in
-`srcQube`. The corresponding RPC policies should obviously be configured
-accordingly.
+When an `allow` action is defined for a target other than `@dispvm`, the target qube is the one given as an argument to `qvm-open-in-vm` in `<SOURCE_QUBE>`. The corresponding RPC policies must be configured accordingly.
 
-Caveat: since there is no user confirmation with `allow`, applications in
-`srcQube` could leak data through URLs or file names. You might notice that an
-URL has been open in the destination qube but it would be too late.
+**Warning:** Since there is no user confirmation with `allow`, applications in `<SOURCE_QUBE>` could leak data through URLs or file names.
 
-### Notes about using disposable qubes and the `@dispvm` keyword in policies
+### Using disposables and the `@dispvm` keyword in policies
 
-It is possible to further restrict a destination DispVM qube by specifying the
-template it is based on with the `@dispvm:templatename` syntax. See the
-[documentation](/doc/how-to-use-disposables/#opening-a-link-in-a-disposable-based-on-a-non-default-disposable-template-from-a-qube)
-for further details and the sample section below.
+It is possible to further restrict a target disposable qube by specifying the template on which it is based with the `@dispvm:<DISPOSABLE_TEMPLATE>` syntax ([learn more](/doc/how-to-use-disposables/#opening-a-link-in-a-disposable-based-on-a-non-default-disposable-template-from-a-qube)).
 
-Caveat: `@dispvm` means "DisposableVMs based on the default DisposableVM
-template of the calling qube", not "*any* DisposableVMs". If you were to run
-`qvm-open-in-vm @dispvm:onlinedvm https://www.qubes-os.org` in `srcQube` and
-`onlinedvm` wasn't the default dvm template for `srcQube`, a policy line with only
-`@dispvm` wouldn't match: it would have to be `@dispvm:onlinedvm`.
-
-If for some reason a user needs to use a disposable qube with a static name -
-which might come handy when using `allow` RPC policies like in the sample
-section below - he/she can do like so (replace `fedora-dvm` with the dvm
-template you want to use):
-
-~~~
-qvm-create --class DispVM --label red --template fedora-dvm qube_name
-~~~
-
-`qube_name` would then be started like a regular qube with the difference that
-like standard disposable qubes its private disk is wiped after each use.
-However, unlike standard disposable qubes, this qube won't "auto power off" when
-the called application (eg. firefox) is closed, so it's up to the user to power
-off (or restart) the qube when he/she deems necessary.
```

Comment:
I think this part is out-of-scope on this page. It belongs on one of the disposable doc pages instead. (If it's already be covered there, then we should [avoid duplicating documentation](https://www.qubes-os.org/doc/documentation-style-guide/#do-not-duplicate-documentation).)